app/src/marketplace/components/Products/settings/index.jsx

```diff
@@ -4,46 +4,13 @@ import React, { type Node } from 'react'
 import get from 'lodash/get'
 
 import NoProductsView from '../NoProductsView'
-import NoMyPurchasesView from '../NoMyPurchasesView'
-import NoMyProductsView from '../NoMyProductsView'
```

Comment:
Looks like removing those effectively makes them generally unused.

<img width="585" alt="Screenshot 2020-01-08 11 00 43" src="https://user-images.githubusercontent.com/320066/71969097-4bbb9a80-3206-11ea-8c7b-f13802a03506.png">

Safe to remove?

Reply:
Yep, removed